src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1717,6 +1718,38 @@ public String getVariableNamePattern() {
 					})
 					.serializer(new EnumSerializer<>(Cat.Type.class)));
 		}
+		
+		EnumUtils<Attribute> attributes = new EnumUtils<>(Attribute.class, "attribute types");
+		Classes.registerClass(new ClassInfo<>(Attribute.class, "attributetype")
+				.user("attribute ?types?")
+				.name("Attribute Type")
+				.description("The attribute type of an entity. Note that this type does not contain any numerical values."
+						+ "See <a href='https://minecraft.gamepedia.com/Attribute#Attributes'>attribute types</a> for more info.")
+				.defaultExpression(new EventValueExpression<>(Attribute.class))
+				.examples(attributes.getAllNames())
+				.since("INSERT VERSION")
+				.parser(new Parser<Attribute>() {
+					@Override
+					@Nullable
+					public Attribute parse(String input, ParseContext context) {
+						return attributes.parse(input);
+					}
+					
+					@Override
+					public String toString(Attribute a, int flags) {
+						return attributes.toString(a, flags);
+					}
+					
+					@Override
+					public String toVariableNameString(Attribute a) {
+						return a.name();
```

Comment:
still not quite right - it should just be return the same thing as `toString(Attribute, int)`  (`attributes.toString(a, flags)`). so just `return toString(a, 0);` will work